tests/plugins/image2/manual/dialogafterresize.md

```diff
@@ -0,0 +1,15 @@
+@bender-tags: 4.12.0, feature, bug, 2672
```

Comment:
After splitting test into two separate ones, declaring them as `feature` & `bug` is no longer needed – one is for feature and one is for bug.